src/vs/editor/contrib/zoneWidget/browser/zoneWidget.ts

```diff
@@ -18,7 +18,6 @@ import { ICodeEditor, IOverlayWidget, IOverlayWidgetPosition, IViewZone, IViewZo
 export interface IOptions {
 	showFrame?: boolean;
```

Comment:
Make sure that references widget, peek view widget, error list widget and breakpoint widget are rendered the same as before.